packages/storybook/src/stories/playground/real-data-demo.stories.tsx

```diff
@@ -1,5 +1,4 @@
 import * as React from 'react';
-import { withA11y } from '@storybook/addon-a11y';
```

Comment:
Does it mean that we should remove `@storybook/addon-a11y` from the list of dependencies?

Reply:
Nope, we just don't need to declare it here